hack/jenkins/common.sh

```diff
@@ -133,8 +133,15 @@ if type -P virsh; then
   virsh -c qemu:///system list --all || true
 fi
 
+
 if type -P vboxmanage; then
   vboxmanage list vms || true
+  # remove inaccessible stale VMs https://github.com/kubernetes/minikube/issues/4872
+  vboxmanage list vms \	
+    | grep inaccessible \	
+    | cut -d'"' -f3 \	
+    | xargs -I {} sh -c "vboxmanage startvm {} --type emergencystop; vboxmanage unregistervm {} --delete" \	
+    || true
   vboxmanage list vms \
     | egrep -o '{.*?}' \
     | xargs -I {} sh -c "vboxmanage startvm {} --type emergencystop; vboxmanage unregistervm {} --delete" \
```

Comment:
Rather than adding a second more-specific command, let's fix the general command that comes afterwards.  For clarity, I recommend splitting it into two commands:

```
vboxmanage list vms | egrep -o '{.*?}' | xargs -I{} vboxmanage startvm {} --type emergencystop || true
vboxmanage list vms | egrep -o '{.*?}' | xargs -I{} vboxmanage unregistervm {} | true
```

I tested this on macOS and it seems to do the right thing.